import-export-cli/cmd/deprecated/importAPI.go

```diff
@@ -84,6 +85,8 @@ func init() {
 		"", "Environment from the which the API should be imported")
 	ImportAPICmdDeprecated.Flags().BoolVar(&importAPICmdPreserveProvider, "preserve-provider", true,
 		"Preserve existing provider of API after importing")
+	ImportAPICmdDeprecated.Flags().BoolVar(&importAPIrotateRevision, "rotate-revision", false, "Rotate the "+
+		"revisions with each update")
```

Comment:
We do not need this in deprecated commands, since those have been deprecated. Isn't it?

Reply:
Fixed with cf58769